### ISSUE TITLE
Elevated circulating thymine concentration

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9558]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
222748 
PMID: 17383919 2/2
see other metabolites

**Your nano-attribution (ORCID)**
